lib/telemetry_metrics.ex

```diff
@@ -487,10 +463,7 @@ defmodule Telemetry.Metrics do
     raise ArgumentError, "expected buckets to be a non-empty list, got #{inspect(term)}"
   end
 
-  @spec metadata_spec_to_function(metadata()) ::
+  @spec tag_values_spec_to_function(tag_values()) ::
```

Comment:
We don’t need this function anymore, right?

Reply:
Ah, yes. This can be removed.